addons/website/views/snippets/s_image_text_overlap.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_image_text_overlap" name="Image - Text Overlap">
+    <section class="s_image_text_overlap pt64 pb64">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="9">
+                <div class="o_grid_item o_grid_item_image order-lg-0 g-col-lg-7 g-height-9 col-lg-7" style="grid-area: 1 / 1 / 10 / 8; --grid-item-padding-x: 0px; z-index: 1; order: 2;">
```

Comment:
Could you remove the vertical padding as well? 
```suggestion
                <div class="o_grid_item o_grid_item_image order-lg-0 g-col-lg-7 g-height-9 col-lg-7" style="grid-area: 1 / 1 / 10 / 8; --grid-item-padding-x: 0px; --grid-item-padding-y: 0; z-index: 1; order: 2;">
```

<img width="623" alt="image" src="https://github.com/user-attachments/assets/55633535-e3da-4895-ade3-ed1669df0703">

Reply:
I wanted to keep it for mobile. Should i use margin classes instead ? 
<img width="591" alt="Screenshot 2024-08-07 at 09 13 29" src="https://github.com/user-attachments/assets/06fe214d-1ef8-4a4b-824b-eb3a2136303d">

Reply:
The enforced `gap: 0` on mobile will be fixed by rd-website.

But yes, I do agree. The visual result would be suboptimal on mobile, while using padding as you did it's still in the range of the editor possibilities and would be editable... eventually, the user will be able to removing this padding in case the the snippet has been dropped before that rd-website patch is merged 🤔

IMO it's ok, thanks... let's just double-check with @sobo-odoo to ensure that will not become a blocking point during the tech. review 👍

Reply:
It's fine either either way, you can keep the padding or remove it, it just depends on what visual you want to achieve :wink: Since there is an option to change the padding, it will never be blocking, and the user will be able to change it :slightly_smiling_face: 

About the gap on mobile, note that it still has to be discussed, I have no idea of when/if it will be implemented, so consider it as impossible for now, like with non-grid mode snippets :+1: (play with the padding if you want some space :wink:)

Some review comments, while I am at it (I took a brief look) :slightly_smiling_face: 
- The commit "tag" should be [IMP] and not [ADD], as [ADD] is for adding new modules (according to the [guidelines](https://www.odoo.com/documentation/master/contributing/development/git_guidelines.html#tag-and-module-name) )
- In the second grid item, the `col-lg` and the `g-col-lg` do not match => they should, so set `col-lg-7` instead of `5` :wink: 
- I don't think you can use the `align-content-center` class, because there is no option to control it and so no way to remove the behavior/reproduce it somewhere else (like if we add a new column in the snippet) => I would say to try using the padding to center it instead (or if really necessary, to set a CSS rule on this snippet grid items, instead of a class on one element, so it applies to every grid items inside it (a bit like with Masonry) :slightly_smiling_face:

Reply:
Thanks @sobo-odoo !
> In the second grid item, the col-lg and the g-col-lg do not match => they should, so set col-lg-7 instead of 5 
Is there a reason that they should match ? I actually want the text element to be smaller. If I set "col-lg-7" it will wrap on a new line

Fixed the rest 👍

Reply:
It is the way the grid sizing option works, it always sync both classes so the templates should reflect  this behavior :wink: 

> If I set "col-lg-7" it will wrap on a new line

Only if you toggle back the normal mode right ? (since in mobile they are one above the other anyway)
=> We don't care what happens when we go back to normal mode, we only made it so the toggle from normal to grid is as close as possible to how it was in normal, but not the other way around => the user only has to resize the columns if he wants both of them to be on the same line :wink:  (as you can see the same goes with the padding: there is no padding when going back to normal mode, because the user only has to set it again)

Reply:
> we only made it so the toggle from normal to grid is as close as possible to how it was in normal, but not the other way around

Why not if by default our snippet is in grid mode and when it seems like using different values is working ? 😄

Reply:
- Normal -> Grid: it is necessary because the grid mode can be toggled when dragging a column, so the grid mode can be activated "implicitely" (like without the used actually wanting to) :+1: Therefore, the layout must stay consistent, as we don't want everything moving randomly (=> bad UX) :wink: 

- Grid -> Normal: the user has to make the decision to go back to normal mode, so it is not appearing unexpectedly, which makes it less important to keep the layout as similar as possible :+1:  Given the way the grid mode works (overlapping items, padding values not specifically multiples of 8, gaps, grid item orders not necessarily in the DOM order after a move), it would have been too much code to manage all that, just to have columns next to each other :shrug: It is simpler to just let the user resize the columns and the paddings and change their order with the arrows (=> it's only a few clicks)
At least with the synced `col-` and `g-col` classes, they at least keep the same size as in grid mode :slightly_smiling_face:

Reply:
Ok I see, thanks for the explanation. Fixed ✅